packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -232,16 +232,17 @@ export class CucumberJSAllureFormatter extends Formatter {
     const labels = this.parseTagsLabels(scenario?.tags || []);
     const links = this.parseTagsLinks(scenario?.tags || []);
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
+    const thread = ALLURE_THREAD_NAME || process?.getuid()?.toString() || process.pid.toString();
```

Comment:
I think we should completely switch to `procces.pid` and drop `getuid`

Reply:
`uid` and `pid` are different things, but yes, better to keep on thing here

Reply:
i think uid don't make sense in that case.

Reply:
when we wank to mark `process id` not `user id`